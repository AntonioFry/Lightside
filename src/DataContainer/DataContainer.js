import React from 'react';
import InfoCard from '../InfoCard/InfoCard';

const DataContainer = ({ peopleData, vehicleData, planetData }) => {
  const allPeople = peopleData.map(person => {
    return <InfoCard 
      personName={person.name}
      birthyear={person.birth_year}
      gender={person.gender}
      height={person.height}
      eyeColor={person.eye_color}
      />
  });
  return (
    <section className='info-card-container'>
      {/* {allPeople} */}
    </section>
  )
}

export default DataContainer;