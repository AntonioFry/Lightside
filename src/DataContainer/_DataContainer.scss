.info-card-container {
  background-color: $primaryColor;
  border: 2px solid $accentColor;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
  min-height: fit-content;
  padding: 10px;
}

h2 {
  font-family: 'STARWARS';
  text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
  color: #f1f1f1
}

.home-prompt {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.home-icon {
  width: 50%;
}