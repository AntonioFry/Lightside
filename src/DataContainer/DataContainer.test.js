import React from 'react';
import DataContainer from './DataContainer/'
import { shallow, mount } from 'enzyme';
import peopleData from '../mockData/peopleData';
import vehiclesData from '../mockData/vehiclesData';
import planetsData from '../mockData/planetsData';



describe('DataContainer', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(
      <DataContainer 
      peopleData={peopleData.results} 
      vehicleData={vehiclesData.results} 
      planetData={planetsData.results}
      favorites={[]}
      favoriteCard={jest.fn()}
      />
    )
    expect(wrapper).toMatchSnapshot();
  })
});