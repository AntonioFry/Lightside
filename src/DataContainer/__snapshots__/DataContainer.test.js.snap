// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataContainer should match the snapshot 1`] = `ShallowWrapper {}`;
